tools/WinObjC.Packaging/Package.Default.props

```diff
@@ -18,7 +18,6 @@
     <_ImportLibsFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\*.lib"/>    
     <_ImportLibsFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\$(Configuration)\*.lib"/>
 
-    <_XamlFilesFromPackage Include="$(MSBuildThisFileDirectory)\xaml\**\*.xaml" />
```

Comment:
><_XamlFilesFromPackage Include="$(MSBuildThisFileDirectory)\xaml\**\*.xaml" /> [](start = 4, length = 78)

Why is this file modified but doesn't have the new .pri file consumption targets?

Package.Default.props is the default consumption props/targets for middleware using WinObjC

If they have xaml components, what needs to happen to ensure it works for them?